src/components/MoviesSearchBar/MoviesSearchBar.scss

```diff
@@ -41,13 +41,13 @@
       padding-left: 2%;
```

Comment:
Whenever possible, please don't use the `%` unit for padding and margins unless really necessary.